src/main/scala/ml/PipelineBuilder.scala

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2017 Michael Matthews.
+// See the LICENCE.txt file distributed with this work for additional
+// information regarding copyright ownership.
+//
+
+package ml
+
+import ml.FeatureEngineering._
+import org.apache.spark.ml.PipelineStage
+import org.apache.spark.ml.feature.VectorAssembler
+
+
```

Comment:
Should be more consistent with spaces between the imports and class/object definition. Will allow for now. Try a linter rule to enforce.